eng/common/scripts/Package-Properties.ps1

```diff
@@ -67,20 +67,26 @@ class PackageProps
     }
 }
 
-# Takes package name and service Name
+# Takes artifact name and service Name
 # Returns important properties of the package as related to the language repo
 # Returns a PS Object with properties @ { pkgName, pkgVersion, pkgDirectoryPath, pkgReadMePath, pkgChangeLogPath }
 # Note: python is required for parsing python package properties.
 function Get-PkgProperties
 {
     Param
     (
-        [Parameter(Mandatory = $true)]
         [string]$PackageName,
+        [string]$ArtifactName,
         [Parameter(Mandatory = $true)]
         [string]$ServiceDirectory
     )
 
+    # This is to help switch PackageName to Artifact name which is more accurate
```

Comment:
What is your transition plan? As we chatted while ArtifactName is the same as the PackageName in some languages it isn't in others. So I like having them be separate values but we have to figure out the correct way to map from one to the other or we have to figure out a good way to add both values to the PackageProperties and allow searching by either.